pdns/dnsdistdist/dnsdist-lua-ffi.hh

```diff
@@ -59,6 +59,15 @@ struct dnsdist_ffi_dnsquestion_t
   boost::optional<std::string> httpScheme{boost::none};
 };
 
+struct dnsdist_ffi_dnsresponse_t
```

Comment:
I think we need to add a `LuaContext::Pusher` specialization for this type, otherwise it's going to be as slow as calling the non-FFI interface.

Reply:
That means we are also going to need to update `LuaFFIResponseAction` and `LuaFFIPerThreadResponseAction`, I'm afraid.

Reply:
Tried to improve that. Hope this will be enough ! Thanks for the review anyway